core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -31,17 +31,20 @@ oppia.animation('.oppia-collection-animate-slide', function() {
 });
 
 oppia.controller('CollectionPlayer', [
-  '$scope', '$http', 'ReadOnlyCollectionBackendApiService',
+  '$scope', '$anchorScroll', '$location', '$http',
+  'ReadOnlyCollectionBackendApiService',
   'CollectionObjectFactory', 'CollectionPlaythroughObjectFactory',
   'alertsService', 'UrlInterpolationService',
-  function($scope, $http, ReadOnlyCollectionBackendApiService,
-    CollectionObjectFactory, CollectionPlaythroughObjectFactory,
-    alertsService, UrlInterpolationService) {
+  function($scope, $anchorScroll, $location,
+           $http, ReadOnlyCollectionBackendApiService,
+           CollectionObjectFactory, CollectionPlaythroughObjectFactory,
+           alertsService, UrlInterpolationService) {
     $scope.collection = null;
     $scope.collectionPlaythrough = null;
     $scope.collectionId = GLOBALS.collectionId;
     $scope.showingAllExplorations = !GLOBALS.isLoggedIn;
     $scope.previewCardIsShown = true;
+    $scope.previewCardMobileIsShown = true;
```

Comment:
Actually, they have separate functions -- the previewCardIsShown in desktop view is for keeping track of whether the blank card is shown (ie. the blank collection card which gets replaced on hovering on the icons with the corresponding lesson tile). Whereas, the previewCardMobileIsShown var is to keep track of whether the current lesson card displayed in mobile view (there is no blank card in mobile view). I guess I should rename it to something else to avoid this confusion. Any suggestions?